app/src/main/java/com/cooper/wheellog/utils/KingsongAdapter.java

```diff
@@ -158,7 +161,23 @@ public boolean decode(byte[] data) {
 
     @Override
     public void updatePedalsMode(int pedalsMode) {
-        WheelData.getInstance().updatePedalsMode(pedalsMode);
+        byte[] data = new byte[20];
```

Comment:
```suggestion
        byte[] data = getEmptyRequest()];
```
чтобы не заполнять первые два и последние три типичных байт